test/source/_plugins/custom_block.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# For testing excerpt handling of custom tags
+
+module Jekyll
+  class DoNothingBlock < Liquid::Block
+  end
+
+  class DoNothingOther < Liquid::Tag
+  end
+end
+
+Liquid::Template.register_tag("do_nothing", DoNothingBlock)
+Liquid::Template.register_tag("do_nothing_other", DoNothingOther)
```

Comment:
These should be under the `Jekyll` namespace..
```ruby
Liquid::Template.register_tag("do_nothing", Jekyll::DoNothingBlock)
Liquid::Template.register_tag("do_nothing_other", Jekyll::DoNothingOther)
```

Reply:
Doesn't putting them in `module Jekyll` automatically do that? When I tested it looked like they were.

Reply:
I've added the test below which seems to confirm that they're in the `Jekyll` namespace.

Reply:
Only if `Liquid` is registering the tag within the `Jekyll` module:
```ruby
module Jekyll
  class DoNothingBlock < Liquid::Block
    # [...]
  end

  Liquid::Template.register_tag("do_nothing", DoNothingBlock)
end

Reply:
I'm so sorry to be dumb, but isn't that what's written above?

Reply:
Do not be sorry. What I see in your plugin is:
```ruby
module Jekyll
  class DoNothingBlock < Liquid::Block
    # [...]
  end
end

Liquid::Template.register_tag("do_nothing", DoNothingBlock)
```

Reply:
Can `assert_match "Jekyll::DoNothingBlock"` pass if it's not in the Jekyll namespace though?

Reply:
It should raise an `uninitialized constant` exception, ideally. But honestly, I do not know why the tests are passing.

I extracted your "plugin" into a freshly generated Jekyll Project and it failed as expected..
:man_shrugging:

Reply:
OK! I'll just set it to be as you suggest :)